### PR TITLE
Add commands to create and list RTDB instances

### PR DESCRIPTION
**Pull request by ryanpbrewster**

### CODE REVIEW

**Comment by bkendall** [comment]
Please re-assign and remove the `DO NOT MERGE` tag after the (internal) API review is approved.

**Comment by ryanpbrewster** [comment]
API proposal was approved, PTAL

File: src/gcp/firedata.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    auth: true,
+    origin: api.firedataOrigin,
+  });
+  if (response.status === 200) {
+    return response.body.instance;
+  }
+  return _handleErrorResponse(response);
+}
+
+export interface DatabaseInstance {
```
Comment (bkendall) [inline src/gcp/firedata.ts]: Put this at the top of the file, please.

File: src/commands/database-instances-create.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Command = require("../command");
+import logger = require("../logger");
+import requirePermissions = require("../requirePermissions");
+import getProjectNumber = require("../getProjectNumber");
+import firedata = require("../gcp/firedata");
+
+export default new Command("database:instances:create <instanceName>")
+  .description("create a realtime database instance")
+  .before(requirePermissions, [])
```
Comment (bkendall) [inline src/commands/database-instances-create.ts]: are there specific permissions required here?

**Comment by bkendall** [comment]
Found the review. LGTM (with nits)

File: src/gcp/firedata.ts (hunk 1 above)
Comment (ryanpbrewster) [reply src/gcp/firedata.ts]: done

File: src/commands/database-instances-create.ts (hunk 2 above)
Comment (ryanpbrewster) [reply src/commands/database-instances-create.ts]: added

**Comment by ryanpbrewster** [comment]
Note that this should probably entail a minor version bump.

**Pull request merged**